doc/authoring_help.md

```diff
@@ -0,0 +1,93 @@
+# Project AZ Help System #
+
+Help authoring for commands is done in a number of places, all of which are contained in the Az code base.  Some help text comes from product code, but it can be overridden using a YAML-based help authoring system.  The YAML-based system is the recommended way to update command and group help text.
+
+## YAML Help Authoring ##
+
+The YAML syntax is described [here](http://www.yaml.org/spec/1.2/spec.html "here").
+
+To override help for a given command:
+
+1. Find the command's module, Example "az account clear"
+	1. Search code base for "account clear"
+	2. Search result: src\command_modules\azure-cli-**profile**\azure\cli\command_modules\**profile**\generated.py
```

Comment:
The backslash is causing some chars to be escaped when the doc is formatted. Consider using / instead as it would align more with Linux users.